scripts/aws-cni-support.sh

```diff
@@ -52,14 +52,14 @@ ip rule show > ${LOG_DIR}/iprule.out
 # iptables-save
 iptables-save > ${LOG_DIR}/iptables-save.out
 
-# iptables -nvL
-iptables -nvL > ${LOG_DIR}/iptables.out
+# iptables -wnvL
+iptables -wnvL > ${LOG_DIR}/iptables.out
```

Comment:
Have you tested that it works? I think you need to give `-w` an argument of how many seconds to wait. I'd guess 1s would be enough, so how about `iptables -w1 -nvL`?

Reply:
"-w" has a default time to wait, but honestly, I am not sure which one, so I agree with your idea.